src/SHADERed/Objects/ProjectParser.cpp

```diff
@@ -1029,7 +1029,7 @@ namespace ed {
 	}
 	std::string ProjectParser::GetProjectPath(const std::string& to)
 	{
-#if defined(_WIN32)
+#if defined(_WIN32) || defined(__APPLE__)
```

Comment:
Nothing in this code path is platform dependent; it's using C++ `std`. Shouldn't this code path work for *all* platforms?
What's the current situation on linux?

Reply:
I'm not testing it on Linux, because I do not have one, but it's indeed should work without any ifdefs.
In my case, I want to be careful and only fix the macOS problem.